src/helpers/helpers.spec.tsx

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
rename file to `functions.spec.tsx` since it only covers tests in that file => same for `styles.spec.tsx`